02.calendar/cal.js

```diff
@@ -1,34 +1,32 @@
 #!/usr/bin/env node
 
 const main = () => {
-  const options = cutInput(process.argv.slice(2))
+  const options = parseArgs(process.argv.slice(2))
   console.log(`      ${options.m}月 ${options.y}`)
-  const date = new Date(Date.UTC(options.y, --options.m, '1'))
+  const date = new Date(Date.UTC(options.y, --options.m))
```

Comment:
```suggestion
  const date = new Date(Date.UTC(options.y, --options.m, 1))
```

とあえて日にちを明示的に指定した方が、JSの仕様にあまり詳しくない人でもコードが理解しやすいかもしれません。
（好みの世界かもしれませんが）

Reply:
修正致しました。
時刻は省略しても、year, month, dayは揃って記述されていたほうが、読みやすいですね。